plugins/experimental/stek_share/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 if(nuraft_FOUND)
   add_atsplugin(stek_share common.cc log_store.cc stek_share.cc stek_utils.cc)
 
-  target_include_directories(stek_share PRIVATE nuraft::nuraft OpenSSL::SSL yaml-cpp::yaml-cpp)
+  target_include_directories(stek_share PRIVATE yaml-cpp::yaml-cpp)
```

Comment:
This is also incorrect. Targets should not be used as arguments to `target_include_directories`. It might need to be `${YAML_INCLUDE_DIR}`.

Reply:
libyaml is broken independent of this fix.

I will have a different PR for libyaml